app/src/main/res/layout/tutorial.xml

```diff
@@ -47,14 +44,31 @@
         android:layout_marginLeft="20dp"
         android:hint="Gangnam-gu, Seoul"
         android:inputType="textCapSentences" />
-
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="horizontal">
```

Comment:
```suggestion
        android:orientation="horizontal"
        android:padding="10dp" >
```

Reply:
And one more suggestion here! 
It looks better if there's a bit of white space.